listenbrainz/webserver/static/js/src/SimilarityScore.tsx

```diff
@@ -19,9 +19,12 @@ const getclassName = (similarityScore: number): string => {
 };
 
 const SimilarityScore = (props: SimilarityScoreProps) => {
-  const { similarityScore, user, type } = props;
-  const className = getclassName(similarityScore);
+  const { user, type } = props;
+
+  let { similarityScore } = props;
```

Comment:
You should probably create a new constant like `roundedSimilarityScore`

Reply:
done